### DISCUSSION TITLE
How to reset statistics

### DISCUSSION THREAD

**Discussion by DPG55** [tnmendes/watt#72] [General]
Cannot find anywhere the process to reset stats. Can anyone help.

**Comment by tnmendes** [comment]
Hey @DPG55, reset stats just work for Kasa Smart devices (not for Tapo) but if you have those devices what you need to do is:

- In the watt app click on the device you want to reset.
- On the device screen click on the top left button
- Now, scroll down until you see the "Reset the stats" button

Here are some print screens:


![IMG_CA9BE58BD475-1](https://user-images.githubusercontent.com/9822076/187900270-cdff648b-216c-4559-8ce2-745177eb10e7.jpeg)
![IMG_7DA73CDE0EB0-1](https://user-images.githubusercontent.com/9822076/187900276-fea1e656-0e32-48d0-bfa0-0ca76eb732a9.jpeg)
![IMG_AB49CC0974F1-1](https://user-images.githubusercontent.com/9822076/187900280-a8cce3ef-e6a3-4c01-a756-65c10fe5d51d.jpeg)

**Comment by DPG55** [comment]
Ok many thanks. A bit disappointing as it's advertised as working with Kasa and Tapo. I've tried unloading the app to reset but that doesn't work either

David G

**Comment by mark98258** [comment]
A master-reset of power consumption for all devices would be more useful along with other all-device edit functions missing from the TP-Link KASA app.
